src/xunit.performance.run/ProgramCore.cs

```diff
@@ -235,11 +239,10 @@ private IEnumerable<PerformanceTestInfo> DiscoverTests(IEnumerable<XunitProjectA
 
                 // Note: We do not use shadowCopy because that creates a new AppDomain which can cause
                 // assembly load failures with delay-signed or "fake signed" assemblies.
-                using (var controller = new Xunit2(
-                    useAppDomain: false,
-                    sourceInformationProvider: new NullSourceInformationProvider(),
+                using (var controller = new XunitFrontController(
                     assemblyFileName: assembly.AssemblyFilename,
                     shadowCopy: false,
```

Comment:
should shadowCopy be ShadowCopy/ShadowCopyOrDefault? I've only glanced at the changes xunit made, so I'm not sure